config/ocp/kustomization.yaml

```diff
@@ -9,7 +9,9 @@ namespace: flotta
 # Note that it should also match with the prefix (text before '-') of the namespace
 # field above.
 namePrefix: flotta-operator-
-
+images:
+- name: "gcr.io/kubebuilder/kube-rbac-proxy:v0.8.0"
+  newName: "registry.redhat.io/openshift4/ose-kube-rbac-proxy:v4.7.0"
```

Comment:
Pls use v4.9.0 which is the OCP version being tested ATM.

Reply:
Done